mmv1/third_party/terraform/services/chronicle/resource_chronicle_watchlist_test.go.tmpl

```diff
@@ -0,0 +1,97 @@
+package chronicle_test
+
+{{- if ne $.TargetVersionName "ga" }}
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccChronicleWatchlist_chronicleWatchlistBasicExample_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"chronicle_id":  envvar.GetTestChronicleInstanceIdFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: acctest.ProtoV5ProviderBetaFactories(t),
+		CheckDestroy:             testAccCheckChronicleWatchlistDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccChronicleWatchlist_chronicleWatchlistBasicExample_basic(context),
+			},
+			{
+				ResourceName:            "google_chronicle_watchlist.example",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"instance", "location", "watchlist_id"},
+			},
+			{
+				Config: testAccChronicleWatchlist_chronicleWatchlistBasicExample_update(context),
+			},
+			{
+				ResourceName:            "google_chronicle_watchlist.example",
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"instance", "location", "watchlist_id"},
+			},
+		},
+	})
+}
+
+func testAccChronicleWatchlist_chronicleWatchlistBasicExample_basic(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+provider "google-beta" {
+ user_project_override = true
+}
+
+resource "google_chronicle_watchlist" "example" {
+ provider = "google-beta"
+ location = "us"
+ instance = "%{chronicle_id}"
+ watchlist_id = "tf-test-watchlist-name%{random_suffix}"
+ description = "tf-test-watchlist-description%{random_suffix}"
+ display_name = "tf-test-watchlist-name%{random_suffix}"
+ multiplying_factor = 1
+ entity_population_mechanism {
+  manual {
+
+  }
+ }
```

Comment:
The indentation should be 2 white spaces.

Reply:
Done!